Revit_Engine/Create/Config/RevitComparisonConfig.cs

```diff
@@ -0,0 +1,73 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Adapters.Revit;
+using BH.oM.Base.Attributes;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.Engine.Adapters.Revit
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Creates a RevitComparisonConfig with specific inputs assigned.")]
+        [Input("parametersToConsider", "Names of the Revit Parameters that will be considered for the comparison." +
+            "By default, this list is empty, so all parameters are considered (except possibly those included in the other property `ParametersExceptions`).")]
+        [Input("propertiesToConsider", "If one or more entries are specified here, only objects/properties that match them will be considered." +
+           "\nE.g. Given input objects BH.oM.Structure.Elements.Bar, specifying `StartNode` will only check that property of the Bar." +
+           "\nSupports * wildcard." +
+           "\nNote that using this will incur in a general slowdown because it is computationally heavy. See the wiki for more details.")]
+        public static RevitComparisonConfig RevitComparisonConfig(List<string> parametersToConsider, List<string> propertiesToConsider = null)
+        {
+            RevitComparisonConfig rcc = new RevitComparisonConfig()
+            {
+                PropertiesToConsider = propertiesToConsider,
+                ParametersToConsider = parametersToConsider
+            };
+
+            return rcc;
+        }
+
+        /***************************************************/
+
+        [Description("Creates a RevitComparisonConfig with specific inputs assigned.")]
+        [Input("parametersToConsider", "Names of the Revit Parameters that will be considered for the comparison." +
+            "By default, this list is empty, so all parameters are considered (except possibly those included in the other property `ParametersExceptions`).")]
+        [Input("considerOnlyParameterDifferences", "If `true`, objects will be considered 'Modified' only if their RevitParameter changed, and only RevitParameterDifferences will be returned.'")]
+        public static RevitComparisonConfig RevitComparisonConfig(List<string> parametersToConsider, bool considerOnlyParameterDifferences = false)
```

Comment:
As above, could we please add consider added/removed/unassigned optional parameters defaulted to `true`?